scripts/snapshot/utils.py

```diff
@@ -23,14 +27,38 @@ def is_snapshot_week(releases):
     return True
 
 
-def get_dependency_version(releases):
+def get_maps_version(releases):
     for release in releases:
-        if is_current_week(release['created_at']) and not is_patch(
-                release['name']) and not ('private' in release['name']):
+        if is_current_week(release['created_at']) \
+                and not is_patch(release['name']) \
+                and not ('private' in release['name']):
             return release['name'].replace('v', '')
     return None
 
 
+def get_nav_native_version(releases):
+    biggest_major = None
```

Comment:
Is there an API in python for sorting where you specify fields with respect to their priority, so that you could specify major, minor, patch ?

Reply:
I'm not sure